spec/models/game_spec.rb

```diff
@@ -43,4 +43,9 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  it "should have correct number of pieces" do
+      game = Game.create
+      game.pieces.count==32
```

Comment:
The standard approach for something like this is to use an RSpec matcher (https://www.relishapp.com/rspec/rspec-expectations/v/3-4/docs/built-in-matchers), giving you `expect(game.pieces.count).to eq(32)`. Your current setup isn't actually generating a valid test and won't fail even if the count is different. You can see this by changing the line to something `game.pieces.count == 2` and running RSpec again.